### PR TITLE
Parameterize business rules

### PR DESCRIPTION
**Pull request by mabe13**
Make it possible to set the business rules as environment variables for the `reset status in` values for default, min, and max.

**Note**: Currently using test values for the environment variables in the makefile profiles.

### CODE REVIEW

**Comment by mabe13** [comment]
> Matt, it all looks good. However, I wonder if we should explicitly name the units we use in the variables. For example drop `VALUE` and include `MINUTES` which results in `RESET_STATUS_IN_MINUTES_DEFAULT`. What do you think?

Yeah that a good idea, might just got with the abbreviated 'MINS' ?

File: application/api/service/serializers/payload_serializer.py [hunk 1]
```diff
@@ -47,13 +51,14 @@ class CapacityStatusRequestPayloadSerializer(serializers.Serializer):
...
-        max_value=1440,
-        help_text="The amount of time, specified in 1 minute blocks up to and including 24 hours (1440 minutes), \
+        default= default_reset_status_in,
+        min_value= min_reset_status_in,
+        max_value= max_reset_status_in,
+        help_text="The amount of time, specified in 1 minute blocks up to and including %d hours (%d minutes), \
             from the time the capacity status is updated by the request to reset the capacity status of the \
-            service back to GREEN. If no value or 0 is provided, the reset time will default to 4 hours \
-            (240 minutes).",
+            service back to GREEN. If no value or %d is provided, the reset time will default to %d hours \
```
Comment (jope1) [inline application/api/service/serializers/payload_serializer.py:59]: For 'If no value or %d is provided' - This is tricky now that we can something other than 0 for the min limit.  When the min limit was zero, we were saying here that we would treat 0 as a non-value, and so therefore we would apply the default.  I don't think we can really do that with a different min limit.  Say our min limit was 10, the help text would say that if 10 is provided we'll apply the default. I don't think we will (or should).  And if 0 is provided, we'll get a validation error. So it's changed the behaviour a bit.  I don't think we can consider 0 as a non-value anymore, and that we'll only default if a value isn't supplied.

Comment (mabe13) [reply application/api/service/serializers/payload_serializer.py:59]: good spot, agree and will remove the special case

Comment (mabe13) [reply application/api/service/serializers/payload_serializer.py]: Could not find the logic for changing the resetStatusIn value to the default value when a request is submitted with it being zero, and have also checked the defaulting doesn't happened for a request against the api. Therefore I have only updated the business logic text.

**Pull request merged**